src/kirby/components/loading-overlay/loading-overlay.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="overlay-wrapper">
+    <div
+    *ngIf="isLoading"
+    [class.backdrop]="showBackdrop"
+    class="spinner-wrapper"
+    background="transparent"
```

Comment:
Could we not move this in to the `scss` file? Just to gemme styling and markup separated.

Reply:
We sure can :)